### ISSUE TITLE
Daydream manifest build error

### ISSUE THREAD

**Issue by nathansizemore** [googlevr/gvr-unity-sdk#489]
When building for Daydream, the following error is received:

sdk  v1.30.0
Unity v 5.6.0b10

```
CommandInvokationFailure: Failed to re-package resources.

~/aapt package --auto-add-overlay \
-v \
-f \
-m \
-J "gen" \
-M "AndroidManifest.xml" \
-S "/Users/nathan/development/radg-19570-realtime-lb/Temp/StagingArea/android-libraries/gvr-permissionsupport-release/res" \
-I "/Users/nathan/development/toolchain/android/platforms/android-25/android.jar" \
-F bin/resources.ap_ \
--extra-packages com.google.gvr.permissionsupport \
-S "/Users/nathan/development/radg-19570-realtime-lb/Temp/StagingArea/android-libraries/gvr-permissionsupport-release/res"

AndroidManifest.xml:4: error: Error: No resource found that matches the given name (at 'theme' with value '@style/VrActivityTheme').
AndroidManifest.xml:5: error: Error: No resource found that matches the given name (at 'enableVrMode' with value '@string/gvr_vr_mode_component').
```

**Comment by nathansizemore** [comment]
Not sure where fault lies. Tried without including the SDK, and the same error appeared, so probably not related to this repo. Unity bug filed [here](https://fogbugz.unity3d.com/default.asp?888439_bhfmicl8f3m4d4fm).

**Comment by nathansizemore** [comment]
Problem only occurs when "VR Supported" is checked in Player Settings. When using the GvrViewerMain prefab, everything works as expected.

**Comment by nathansizemore** [comment]
Issue has been resolved with latest version of Unity beta 5.6.0b11.
